tap_hubspot_beta/streams.py

```diff
@@ -873,6 +873,74 @@ def post_process(self, row, context):
 
         return None
 
+class ArchivedProductsStream(hubspotV3Stream):
+    """Archived Products Stream"""
+
+    name = "products_archived"
+    path = "crm/v3/objects/companies?archived=true"
```

Comment:
Why is the endpoint you're using here `companies`? We are meant to be fetching `products`